Gating Group Collections (#1525)
huggingface/hub-docs: docs/hub/collections.md

Set your collection to **private** if you don't want it to be accessible via its URL (it will not be displayed on your profile/organization page). For organizations, private collections are only available to members of the organization.

> ### Gating Group Collection (Enterprise only)

Reviewer: ```suggestion
### Gating Group Collection (Enterprise Hub)
```

Maybe, to make it consistent with the title that appears in datasets-gated.md

> For [Enterprise](https://huggingface.co/docs/hub/en/enterprise-hub) subscribers, you can gate all the repos in a collection, enabling **access requests** for all models and datasets in it.

Reviewer 2: ```suggestion
For [Enterprise Hub](https://huggingface.co/docs/hub/en/enterprise-hub) subscribers, you can gate all the repos in a collection, enabling **a single access request** for all models and datasets in it.
```

> This feature allows you to restrict access to specific collections: users must agree to share their contact information (username and email address) with the organization to access the models and datasets files when enabled.

Reviewer: ```suggestion
This feature allows you to restrict access to specific collections: users must agree to share their contact information (username and email address) with the organization to access the models and datasets in a gating group collection.
```

> Please note that this feature is only available to Enterprise subscribers: more information about Gating Group Collection can be found in [our doc](https://huggingface.co/docs/hub/en/enterprise-hub-gating-group-collections).

Reviewer: ```suggestion
Please note that this feature is only available to Enterprise Hub subscribers: more information about Gating Group Collection can be found in [our doc](https://huggingface.co/docs/hub/en/enterprise-hub-gating-group-collections).
```
